deltachat-ffi/deltachat.h

```diff
@@ -2567,6 +2567,25 @@ dc_lot_t*        dc_chatlist_get_summary     (const dc_chatlist_t* chatlist, siz
 dc_context_t*    dc_chatlist_get_context     (dc_chatlist_t* chatlist);
 
 
+/**
+ * Get info summary for a chat, in json format. 
+ *
+ * The returned json string has the following key/values: 
+ *
+ * id: chat id 
+ * name: chat/group name 
+ * color: color of this chat 
+ * last-message-from: who sent the last message 
+ * last-message-text: message (truncated) 
+ * last-message-state: DC_STATE* constant
+ * last-message-date: 
+ * avatar-path: path-to-blobfile 
+ * is_verified: yes/no
+
+ * @return a utf8-encoded json string containing all requested info. Must be freed using dc_lot_unref().  NULL is never returned.
+ */
+char*            dc_chat_get_info_json       (dc_context_t* context, size_t chat_id);
```

Comment:
the function name should not start with `dc_chat_*` if it shall be compatible to the existing api.

the function names start with the name of the object it takes as the first parameter, with the exception that `dc_context_` is shortcutted by `dc_`.

so, the name should be `dc_get_chat_info_json` or the first parametr should be a `dc_chat_t`.